### PR TITLE
Restructure hello-world examples to standardize for tests

### PR DESCRIPTION
**Pull request by nvkevlu**
Restructures the hello examples to have the job in a job_configs folder within each example folder. Using requirements.txt to standardize the necessary requirements for each example.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

**Comment by nvkevlu** [comment]
> Question: The design change the <example_name> to <example_name>/job_configs/<job_1>
> 
> can we simply called <example_name> to <example_name>/jobs/<job_1>
> 
> the directory actually consists both job configurations as well as custom code, so it shouldnt be called job_configs

This was what it was in cifar10 and prostate examples, so we should probably standardize it throughout if we decide on this.

File: examples/hello-world/hello-cyclic/README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ pip3 install tensorflow
 Use nvflare simulator to run the hello-examples:
 
 ```
-nvflare simulator -w /tmp/nvflare/ -n 2 -t 2 hello-cyclic
+nvflare simulator -w /tmp/nvflare/ -n 2 -t 2 hello-cyclic/jobs/hello-cyclic
```
Comment (YuanTingHsieh) [inline examples/hello-world/hello-cyclic/README.md:22]: hello-cyclic needs additional requirements `tensorflow`.

We should add a `requirements.txt` under hello-cyclic that include this^

File: examples/hello-world/hello-tf2/README.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ pip3 install tensorflow
 Use nvflare simulator to run the hello-examples:
 
 ```
-nvflare simulator -w /tmp/nvflare/ -n 2 -t 2 hello-tf2
+nvflare simulator -w /tmp/nvflare/ -n 2 -t 2 hello-tf2/jobs/hello-tf2
```
Comment (YuanTingHsieh) [inline examples/hello-world/hello-tf2/README.md:21]: hello-tf2 needs additional requirements tensorflow.

We should add a requirements.txt under hello-tf2 that include this^

**Review by YuanTingHsieh — CHANGES_REQUESTED** [review]
Thanks for the effort, mostly looks good, only 2 examples missing requirements.txt

File: examples/hello-world/hello-cyclic/README.md (hunk 1 above)
Comment (nvkevlu) [reply examples/hello-world/hello-cyclic/README.md:22]: Added

File: examples/hello-world/hello-tf2/README.md (hunk 2 above)
Comment (nvkevlu) [reply examples/hello-world/hello-tf2/README.md:21]: Added

**Review by YuanTingHsieh — APPROVED** [review]
Thanks for the effort, LGTM.

**Pull request merged**